app-modules/alert/src/Filament/RelationManagers/AlertsRelationManager.php

```diff
@@ -23,6 +27,17 @@ class AlertsRelationManager extends RelationManager
 {
     protected static string $relationship = 'alerts';
 
+    public static function getBadge(Model $ownerRecord, string $pageClass): ?string
+    {
+        /** @var Student|Prospect $ownerRecord */
+        $alertCount = Cache::tags('alert-count')->remember('alert-count-' . $ownerRecord->id, now()->addMinutes(5), function () use ($ownerRecord) {
+            // TODO: When it is decided how alerts are "resolved" this will need to take that into account to only display unresolved alerts
+            return $ownerRecord->alerts()->count();
+        });
```

Comment:
```suggestion
        $alertCount = Cache::tags('alert-count')
            ->remember(
                'alert-count-' . $ownerRecord->id,
                now()->addMinutes(5),
                function () use ($ownerRecord): int {
                    // TODO: When it is decided how alerts are "resolved" this will need to take that into account to only display unresolved alerts
                    return $ownerRecord->alerts()->count();
                });
```

Also, do we prefer the usage of Facades or helpers() in this app?